pkg/dbtutil/dbt_util.go

```diff
@@ -0,0 +1,103 @@
+package dbtutil
```

Comment:
IMO this file doesn't justify a top-level package.
Maybe it can go under metastore > dbt > utils.go.
@nopcoder may have an opinion.